NVIDIA-Driver/install.bash

```diff
@@ -54,6 +55,56 @@ Section "Files"
         ModulePath      "/opt/nvidia/lib64/xorg/modules"
 EndSection
 EOF
+cat <<EOF | sudo tee /etc/X11/xorg.conf.d/10-nvidia-drm-outputclass.conf > /dev/null
+# This xorg.conf.d configuration snippet configures the X server to
+# automatically load the nvidia X driver when it detects a device driven by the
+# nvidia-drm.ko kernel module.  Please note that this only works on Linux kernels
+# version 3.9 or higher with CONFIG_DRM enabled, and only if the nvidia-drm.ko
+# kernel module is loaded before the X server is started.
+Section "OutputClass"
+    Identifier  "intel"
+    MatchDriver "i915"
+    Driver      "modesetting"
+EndSection 
+
+Section "OutputClass"
+    Identifier      "nvidia"
+    MatchDriver     "nvidia-drm"
+    Driver          "nvidia"
+    Option          "AllowEmptyInitialConfiguration"
+    Option          "PrimaryGPU" "yes"
+    ModulePath      "/opt/nvidia/lib64/xorg/modules"
+    ModulePath      "/usr/lib64/xorg/modules"
+EndSection
+EOF
+cat <<EOF | sudo tee /etc/X11/xorg.conf.d/90-mwhd.conf > /dev/null
+Section "Module"
+	Load "modesetting"
+EndSection
+
+Section "Device"
+	Identifier "nvidia"
+	Driver "nvidia"
+	BusID "PCI:1:0:0"
+	Option "AllowEmptyInitialConfiguration"
+EndSection
+EOF
+
+# Optimus workaround (maybe should ask user first?)
+echo -e "\e[33m\xe2\x8f\xb3 Applying Optimus workarounds...\e[m"
+if [ ! -d /usr/local/share ]; then
+  sudo mkdir -p /usr/local/share
+fi
+cat <<EOF | sudo tee /usr/local/share/optimus.desktop > /dev/null
+[Desktop Entry]
+Type=Application
+Name=Optimus
+Exec=sh -c "xrandr --setprovideroutputsource modesetting NVIDIA-0; xrandr --auto"
+NoDisplay=true
+X-GNOME-Autostart-Phase=DisplayServer
+EOF
+sudo ln -s /usr/local/share/optimus.desktop /usr/share/gdm/greeter/autostart/optimus.desktop
+sudo ln -s /usr/local/share/optimus.desktop /etc/xdg/autostart/optimus.desktop
```

Comment:
I prefer not to add this part for two reasons:
1. `Optimus` support isn't reliable. Even NVIDIA itself suggests that if you cannot turn off integrated GPU this may not work.
2. I cannot test this.

Reply:
I have never ever been able to boot CL without this specific workaround, even when disabling iGPU (via the kernel command line arguments). Furthermore it never caused any trouble, so I don't think it will affect non-optimus devices...

Reply:
line 106: This is not halal because it's modifying `/usr/share`. The solution is to do something similar to line 31 to 38 in `pre_install.bash`.

line 107: Make sure `/etc/xdg/autostart/` exists before creating that symlink